### ISSUE TITLE
Stage Display Color

### ISSUE THREAD

**Issue by 1024jon** [jswetzen/ppstagedisplay#6]
This project is awesome, I got it up and running on a Pi in no time and so far it is working great.  Just one thought, is it a possibility to have it pull the color info from what is configured in the PP stage display configuration?  Just a curiosity.  Thanks!

**Comment by kevindoveton** [comment]
should be fixed in pull request #8

**Comment by jswetzen** [comment]
Thank you both for the input! Good idea and a good solution!
@1024jon, did you get the code or the raspberry pi image? Because I won't be able to make a new image right now.

**Comment by 1024jon** [comment]
Thats awesome, I will test it out today.  I just pulled the code.  For anyone interested, I ran "curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -" from this guide [Here](http://thisdavej.com/beginners-guide-to-installing-node-js-on-a-raspberry-pi/) to install NodeJS, then just cloned and ran node index.js and it worked first try.  Then I scripted chrome and ppstagedisplay to run on boot.  Thanks guys, this rocks!
